Description: remove meinheld and bjoern dependencies
 since they are not available as a debian package yet
Author: David Douard <david.douard@logilab.fr>
Bug-Debian: https://bugs.debian.org/784654

Index: chaussette/chaussette/tests/test_backend.py
===================================================================
--- chaussette.orig/chaussette/tests/test_backend.py
+++ chaussette/chaussette/tests/test_backend.py
@@ -8,12 +8,12 @@ from chaussette.backend import backends
 
 IS_PYPY = hasattr(sys, 'pypy_version_info')
 
-PY2 = ['bjoern', 'eventlet', 'fastgevent', 'gevent',
-       'geventwebsocket', 'geventws4py', 'meinheld',
+PY2 = ['eventlet', 'fastgevent', 'gevent',
+       'geventwebsocket', 'geventws4py',
        'socketio', 'tornado', 'waitress',
        'wsgiref']
 PYPY = ['tornado', 'waitress', 'wsgiref']
-PY3 = ['meinheld', 'tornado', 'waitress', 'wsgiref']
+PY3 = ['tornado', 'waitress', 'wsgiref']
 
 
 class TestBackend(unittest.TestCase):
Index: chaussette/setup.py
===================================================================
--- chaussette.orig/setup.py
+++ chaussette/setup.py
@@ -30,13 +30,13 @@ tests_require = ['nose', 'waitress', 'to
                  'requests', 'minimock']
 
 if not PYPY:
-    tests_require += ['meinheld', 'greenlet']
+    tests_require += ['greenlet']
 
 if sys.version_info[0] == 2:
     tests_require += ['PasteDeploy', 'Paste', 'unittest2', 'ws4py']
     if not PYPY:
         tests_require += ['gevent', 'gevent-websocket', 'eventlet',
-                          'gevent-socketio', 'bjoern']
+                          'gevent-socketio']
 
 setup(name='chaussette',
       version=__version__,
